package.json

```diff
@@ -12,7 +12,8 @@
     "e2e": "ng e2e",
     "build-watch": "ng build --watch",
     "deploy": "rsync -avhe ssh ./dist/* --delete $SSH_ACCOUNT:$DEPLOY_PATH",
-    "sync": "./node_modules/.bin/watch --wait=3 \"echo Account: $SSH_ACCOUNT && npm run deploy && echo 'App is synced!'\" dist"
+    "sync": "./node_modules/.bin/watch --wait=3 \"echo Account: $SSH_ACCOUNT && npm run deploy && echo 'App is synced!'\" dist",
+    "contributors:add": "npx all-contributors add allanbowe code,test,review,video,doc && npx all-contributors add yuryshkoda code,test,projectManagement,video,doc && npx all-contributors add krishna-acondy code,test,review,infra,platform,maintenance,content && npx all-contributors add saadjutt01 code,test,review,mentoring,doc && npx all-contributors add sabhas code,test,review,ideas && npx all-contributors add medjedovicm code,test,review,infra && npx all-contributors add vladislavparhomchik test,review"
```

Comment:
I'm not sure we need this command?  It's a one time thing (not something we'd re-use) unless I'm mistaken

Reply:
Ok, I will delete then.